### ISSUE TITLE
Browser extension (non-production) icon is not visible in high contrast "High Contrast Black" mode

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-web#5801]
## Describe the bug

This is closely related to #4516, which was recently fixed for prod. Unfortunately, that change included only the prod icon and left the other icons untouched. 

Can we possibly get new icons for the non-production icons?

**To Reproduce**

1. Install a non-prod version of the web extension
2. Pin them to the command bar
3. Enable High Contrast Black vis settings
4. Look at the icons--if ambitious, run color contrast tools

## Screenshots

What it look like (zoomed) in HC black when I have all 3 non-prod versions:

![image](https://user-images.githubusercontent.com/45672944/180895234-4b64cd2c-a317-46df-948c-a9d8576ae2f6.png)

It's almost as bad in dark mode (non-high contrast), especially Insider mode if there's a mouse hover, as shown here:

![image](https://user-images.githubusercontent.com/45672944/180896297-091a3fb2-8460-433c-a545-3e459895256e.png)

## Context (please complete the following information)

- OS Name & Version: Windows 10 21H2
- AI-Web Version & Environment: Current bits
- Browser Version: Chrome 103.0.5060.134

**Comment by ferBonnin** [comment]
we need to get new images from design team for these icons too

**Comment by ferBonnin** [comment]
for consistency, lets also ask for playground (if needed)

**Comment by DaveTryon** [comment]
@nang4ally, we just need to get revised artwork for Insider, Canary, and Dev artwork. We did this for prod with #4516. You can see exactly what the icon changes were at https://github.com/microsoft/accessibility-insights-web/pull/5652/files

**Comment by DaveTryon** [comment]
This was released with 2.39.0
